packages/libs/src/services/web3Manager/Web3Manager.ts

```diff
@@ -409,7 +419,7 @@ const ethSignTypedData = async (
       // fix per https://github.com/ethereum/web3.js/issues/1119
     }
 
-    ;(web3.currentProvider as HttpProvider).send(
+    ; (web3.currentProvider as HttpProvider).send(
```

Comment:
Sup with this?

Reply:
formatter I think, i'll run the linter to remove it